src/api/Feed.js

```diff
@@ -184,14 +185,80 @@ export const getParsedFileActivitiesResponse = (response?: { entries: FileActivi
                     return { ...source[FILE_ACTIVITY_TYPE_APP_ACTIVITY] };
                 }
 
+                case FILE_ACTIVITY_TYPE_VERSION: {
+                    // $FlowFixMe
```

Comment:
It was creating a lot of errors, because its not expecting a versions item with these fields from UAA. I was able to add some of the fields from UAA into the version type, so I'll remove this now.